lms/djangoapps/ccx/utils.py

```diff
@@ -260,3 +261,83 @@ def is_email(identifier):
     except ValidationError:
         return False
     return True
+
+
+def add_master_course_staff_to_ccx(course, ccx_key, display_name):
+    """
+    Added staff role on ccx to all the staff members of master course.
+
+    Arguments:
+        course (CourseDescriptorWithMixins): Master course instance
+        ccx_key (CCXLocator): CCX course key
+        display_name (str): ccx display name for email
+    """
+    list_staff = list_with_level(course, 'staff')
+    list_instructor = list_with_level(course, 'instructor')
+
+    with ccx_course(ccx_key) as course:
```

Comment:
@doctoryes  these two are one same course.
```with ccx_course(ccx_key) as course:``` means overridden course in which content of some fields is override. We access overridden course with special id call ``CCXLocator``.

@pdpinch correct me if i am wrong?